blog/docs/events/security-audit-2023.md

```diff
@@ -0,0 +1,25 @@
+
+Knative Extensions are optional plugins and extensions for different Knative use cases. For example, users can find different officially maintained integrations for Knative Eventing, and the Security-Guard project also resides in Knative Extensions. Many Knative Extensions subprojects are not fully mature yet, with many having Alpha and Beta release statuses. 
+
+Ada Logics found 16 security issues of which all except for one have been fixed with upstream patches. Three of the found issues are in third-party dependencies to Knative. One of the issues found is scored with high severity due to its potential impact (ADA-KNATIVE-23-7). However, an attacker would need to compromise the Knative user’s supply-chain, which is highly unlikely to occur in a real-world scenario because few Knative users consume Knative in a way that would enable the attack. In addition, the attacker would need to time their attack to a high degree. The issue has been fixed.
+
+One CVE was assigned during the audit for a vulnerability that could allow an attacker with already escalated privileges to cause further damage in the cluster. The attacker needs to first establish a position in a Knative pod, and from there, they could exploit the vulnerability and cause denial of service of the Knative autoscaling, thereby denying any autoscaling of Knative. The issue was assigned CVE-2023-48713 of Moderate severity and has been fixed in v1.10.5, v1.12.0 and v1.11.3.
+
+The auditors found that Knative does not include provenance with releases; Provenance is a critical component of complying with [SLSA](https://slsa.dev/)  and ensuring tamper resistance of release artifacts. Recently, the SLSA community released v1.9.0 of the [slsa-github-generator](https://github.com/slsa-framework/slsa-github-generator) which produces SLSA L3 compliant provenance. slsa-github-generator ensures tamper-resistance of artifacts by producing verifiable provenance, thereby mitigating a series of known supply-chain risks, many of which have been exploited in the wild in recent years. Ada Logics recommend that Knative switches its build to the slsa-github-generator to comply with SLSA L3.
+
+Knative would like to thank Ada Logics for conducting the security audit, OSTIF for facilitating it and the CNCF for funding the audit.
+
+Knative is an open-source project that encourages community contributions. If you wish to contribute to Knative’s ongoing security work, we recommend that you read through the report, in particular the strategic recommendations and the SAST tooling section, which both include practical recommendations for Knative’s security posture. In addition, work with adopting the slsa-github-generator is an excellent task for anyone who wishes to improve Knative’s supply-chain risk mitigation. If you have questions, comments or suggestions, you can [engage with the community in several ways](https://knative.dev/docs/community/).
```

Comment:
Given that we're not using GitHub Actions to build our releases, I'm not sure we should steer people towards the `slsa-github-generator`.  (If we were, that would be a great tool.)

```suggestion
Knative is an open-source project that encourages community contributions. If you wish to contribute to Knative’s ongoing security work, we recommend that you read through the report, in particular the strategic recommendations and the SAST tooling section, which both include practical recommendations for Knative’s security posture. If you have questions, comments or suggestions, you can [engage with the community in several ways](https://knative.dev/docs/community/).
```